arangod/Cluster/ClusterInfo.cpp

```diff
@@ -229,6 +241,48 @@ void ClusterInfo::cleanup() {
   theInstance->_currentCollections.clear();
 }
 
+void ClusterInfo::triggerBackgroundGetIds() {
+  // Trigger a new load of batches
+  _uniqid._nextBatchStart = 1ULL;
+  _uniqid._nextUpperValue = 0ULL;
+
+
+  try {
+    if (_uniqid._backgroundJobIsRunning) {
+      return ;
+    }
+    _uniqid._backgroundJobIsRunning = true;
+    std::thread([this]{
```

Comment:
Why aren't we using the Scheduler here?
We should not have all the different features start their own threads, because this is completely out of control.

Reply:
No, using the scheduler would be very bad here, since it could have a long queue, maybe even one which whose entries depend on a new uniqId being found. This could produce deadlock and slow down things seriously.